nmrfx-analyst-gui/src/main/resources/fxml/NMRScene.fxml

```diff
@@ -40,15 +45,15 @@
                     <children>
                         <VBox fx:id="btoolVBox">
                             <children>
-                                <Pane HBox.hgrow="ALWAYS"/>
+                                <Pane HBox.hgrow="ALWAYS" />
                             </children>
                         </VBox>
                     </children>
                 </VBox>
             </bottom>
         </BorderPane>
-        <Separator orientation="VERTICAL"/>
-        <StackPane fx:id="processorPane" VBox.vgrow="ALWAYS"/>
+        <Separator orientation="VERTICAL" />
+      <NMRControlRightSidePane fx:id="nmrControlRightSidePane" />
```

Comment:
So we can create custom components within the fxml using another fxml + java combination, the only issue is that we have to build the analyst-gui jar file and add it to scene builder if we want to be able to load the fxml files containing our custom component in scene builder. Its easy enough to do but maybe not very clear for someone that they would need to do that

Alternatively, I could have created NMRControlRightSidePane only in a java file, and added it to the scene in the FXMLController.java once the NMRScene.fxml file was loaded, but it cannot be included within the NMRScene.fxml file in that case.

Reply:
not doing it this way anymore